modules/viewing-triggered-pipeline-runs.adoc

```diff
@@ -33,7 +33,7 @@ The *Run details* page opens.
 A table opens that shows list of triggered runs. After a run has completed its execution, the run's status is displayed in the *Status* column in the table, indicating whether the run has succeeded or failed.
 
 .Verification
-* A list of previously triggered runs are displayed in the *Triggered* tab on the *Runs* page.
+* A list of previously triggered runs display in the *Triggered* tab on the *Runs* page.
```

Comment:
Going by IBM style, looks like this should be 'are displayed'.

Reply:
Fixed.